docs/community.md

```diff
@@ -78,7 +84,7 @@ If you have a project you want featured on the community site, [reach out](mailt
 |------|------|
 | <center>[MessagePack Debugger](https://github.com/algorand/messagepack-debugger)</center> | View and compare messagepack encoded objects. |
 | <center>[PyTeal](https://github.com/algorand/pyteal)</center> | PyTeal is a Python language binding program for ASC1. PyTeal allows developer to express TEAL-based smart contract logic using Python. This dev tool abstracts TEAL and does type-checking at contraction time.|
-| <center>[Reach](https://reach.sh)</center> | Write complex smart contracts in a JS-like language. Currently does not target MainNet / TestNet, but you can already test locally with ease. See blog update [here](https://medium.com/reach-sh/reach-september-update-7d2bb04bcd7e).  |
+| <center>[Reach](https://reach.sh)</center> | Write complex smart contracts in a JS-like language. Currently does not target MainNet / TestNet, but you can already test locally with ease. See docs [here](https://docs.reach.sh/).  |
```

Comment:
You can target MainNet and TestNet at this time. Reword this to something like:
 
Write complex smart contracts in a JS-like language. Deploy to MainNet, TestNet or build and test locally with ease. See docs [here](https://docs.reach.sh/).

Reply:
updated. Nice catch.